build.sh

```diff
@@ -60,6 +60,10 @@ case "$TRAVIS_OS_NAME" in
 
     echo 'Install npm packages for Linux'
     npm install --save-dev electron-installer-debian --silent
+    npm install -g --save-dev electron-installer-redhat --silent
```

Comment:
Why is redhat installed globally but debian isn't?

Is the global flag needed?

Reply:
I left old code working as is and so didn't fuck with it, but I can take a look.

Reply:
Actually that might be an issue so I'll test it on Travis to see how it goes. Perhaps it doesn't work on a clean run (i.e. continuous integration).